specreduce/utils/utils.py

```diff
@@ -0,0 +1,181 @@
+import copy
+
+import numpy as np
+
+from specreduce.core import _ImageParser
+from specreduce.tracing import Trace, FlatTrace
+from specreduce.extract import _ap_weight_image
+
+
+def measure_cross_dispersion_profile(image, trace=None, crossdisp_axis=0,
+                                     width=None, pixel=None, pixel_range=None,
+                                     statistic='median'):
```

Comment:
is this intended to be public immediately or are we keeping it internal for now?  Should it be added to `__all__` and the docs if it is public?

Reply:
i don't see why it can't be public, ill add it. 

in terms of making this part of the extraction steps, i think it can still live in 'utils' as a standalone and we can add methods to the steps to call it if that would be helpful. should that be done in this ticket or a follow up?